server/constants.py

```diff
@@ -5,6 +5,13 @@
 STAFF_ROLE = 'staff'
 INSTRUCTOR_ROLE = 'instructor'
 LAB_ASSISTANT_ROLE = 'lab assistant'
+ROLES_NEW={
```

Comment:
`ROLES_NEW` is not a very descriptive name. `ROLE_DISPLAY_NAMES` would be better